src/commcare_cloud/ansible/roles/couchdb2/tasks/main.yml

```diff
@@ -91,6 +92,10 @@
     status_code: 201, 409 # 409 means already set up (conflict)
     body: '{}'
   when: inventory_hostname == groups.couchdb2.0 and item != groups.couchdb2.0
+  register: result
+  until: not result.failed or 'Connection refused' not in result.msg
+  retries: 5
+  delay: 10
```

Comment:
Copied from the retry configuration in the following task. Though chosen a shorter waiting time since 20 seconds seemed a bit of overkill.